packages/perseus/src/types.ts

```diff
@@ -147,7 +147,7 @@ export type APIOptions = Readonly<{
     readOnly?: boolean;
     answerableCallback?: (arg1: boolean) => unknown;
     getAnotherHint?: () => unknown;
-    interactionCallback?: () => unknown;
+    interactionCallback?: (widgetData: {[widgetId: string]: any}) => void;
```

Comment:
I feel like this type will be improved as part of of LC-1756 when we implement Kevin's typing solution.

Reply:
That's next on the docket?